skimage/exposure/tests/test_exposure.py

```diff
@@ -726,10 +732,10 @@ def test_adjust_inv_sigmoid_cutoff_half():
     assert_array_equal(result, expected)
 
 
-def test_negative():
+def test_adjust_sigmoid_negative():
```

Comment:
```suggestion
@pytest.mark.parametrize(
    'exposure_function',
    [exposure.adjust_gamma, exposure.adjust_sigmoid, exposure.adjust_log]
)
def test_adjust_negative(exposure_function):
```

adjust_gamma will also raise a ValueError in this case for float inputs, so shouldn't we keep that test case as well? It looks like adjust_log is a third case that would raise a ValueError here so perhaps just parameterize this test as suggested here?

Reply:
Good point! It's just that the current tests are organized into:

* Test Gamma Correction (line 534)
* Test Logarithmic Correction (line 601)
* Test Sigmoid Correction (line 647)

So, if we go for the parameterized version, maybe we should move this test right before or right after these 'sections.'